### PR TITLE
kubexit: Init at 0.3.2

### PR DESCRIPTION
**Pull request by qjoly**
###### Description of changes

Add kubexit 

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by wegank — APPROVED** [review]

File: pkgs/applications/networking/cluster/kubexit/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    hash = "sha256-Kzom+/Xad6SI9czw4xvmTbJ+bNB9mF2oSq37IFn384U=";
+  };
+
+  vendorHash = "sha256-RA3+S5Pad+4mNUgcZ2Z0K0FKA3Za5o1ko049GM4yQQ8=";
+  ldflags = [ "-s" "-w" ];
+
+  doCheck = true;
+
+  meta = with lib; {
+    description = ''Artificial Intelligence Infrastructure-as-Code Generator.'';
```
Comment (wegank) [inline pkgs/applications/networking/cluster/kubexit/default.nix:20]:
```suggestion
    description = "Artificial Intelligence Infrastructure-as-Code Generator";
```

**Comment by QJoly** [comment]
I squash the commit tomorrow (it's 10PM in my country)

File: pkgs/applications/networking/cluster/kubexit/default.nix [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-Kzom+/Xad6SI9czw4xvmTbJ+bNB9mF2oSq37IFn384U=";
+  };
+
+  vendorHash = "sha256-RA3+S5Pad+4mNUgcZ2Z0K0FKA3Za5o1ko049GM4yQQ8=";
+  ldflags = [ "-s" "-w" ];
+
+  doCheck = true;
+
```
Comment (wegank) [inline pkgs/applications/networking/cluster/kubexit/default.nix:18]:
```suggestion

```

File: pkgs/applications/networking/cluster/kubexit/default.nix [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    owner = "karlkfi";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-Kzom+/Xad6SI9czw4xvmTbJ+bNB9mF2oSq37IFn384U=";
+  };
+
+  vendorHash = "sha256-RA3+S5Pad+4mNUgcZ2Z0K0FKA3Za5o1ko049GM4yQQ8=";
+  ldflags = [ "-s" "-w" ];
+  meta = with lib; {
+    description = "Command supervisor for coordinated Kubernetes pod container termination.";
```
Comment (wegank) [inline pkgs/applications/networking/cluster/kubexit/default.nix:17]:
```suggestion
    description = "Command supervisor for coordinated Kubernetes pod container termination";
```

**Pull request merged**